schemas/validation_tests/individual-valid-1.json

```diff
@@ -0,0 +1,41 @@
+{
+    "object_id": {
+        "alias" : "NM305004", 
+        "center_name" : "EBI-TEST",
+        "ega_accession": "EGAI00000000001"
+    },
+    "minimal_public_attributes": {
+        "subject_id": "donor-NM305004",
+        "biological_sex": "male",
+        "phenotype": {
+            "experimental_condition_term": "fibroadenoma",
+            "experimental_condition_curie": "EFO:1000254"
+        }
+    },
+    "individual_relationships": [
+        {
+            "r_type": {
+                "r_type_id": "develops_from"
+            },
+            "r_source": {
+                "object_id": {
+                    "ega_accession": "EGAN00000000001"
+                },
+                "object_type": "sample"
+            },
+            "r_target": {
+                "object_id": {
+                    "alias": "NM305004",
+                    "center_name": "EBI-TEST"
+                },
+                "object_type": "individual"               
+            }
+        }
+    ],
+    "individual_attributes": [
```

Comment:
The plan is that all metadata added here under `individual_attributes` will be private/controlled access, right? Because potentially identifying information could be added here? This would be great as it gives us a very clear definition of what is publicly searchable (under `minimal_public_attributes`) and what is controlled access (under `individual_attributes`).

Reply:
Indeed, everything within the individual besides the object's ``core_id`` (alias, accession, etc.) and the ``minimal_public_attributes`` would be private.